Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ruby:3.1.2
 EXPOSE 4567
 CMD ["/usr/bin/supervisord"]
 
-RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip -y
+RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip -y && apt-get install jq -y
```

Comment:
Can just add jq after pip and avoid another &&

Reply:
ahh ya true!
```suggestion
RUN apt-get update && apt-get install --no-install-suggests -y supervisor && apt-get install pip jq -y
```